packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -495,7 +495,15 @@ export const TracksTable = ({
       if (!isLocked || deleted || isOwner || !isPremiumEnabled) {
         return null
       }
+
       if (isContentUSDCPurchaseGated(track.stream_conditions)) {
+        // Format the price as $$ with 2 digit decimal cents
+        const formattedPrice = (
```

Comment:
do we not have a utility in common for something like this? might be worth adding if so?

Reply:
Lemme check

Reply:
Ah looks like we do